src/app/zap-templates/zcl/data-model/chip/window-covering.xml

```diff
@@ -23,14 +23,9 @@ limitations under the License.
     <server tick="false" init="false">true</server>
     <tag name="LF" description="Lift Control"/>
     <tag name="TL" description="Tilt Control"/>
-    <tag name="PA" description="Position Aware"/>
+    <tag name="PA_LF" description="Position Aware lift control"/>
     <tag name="ABS" description="Absolute Positioning"/>
-    <globalAttribute side="server" code="0xFFFC" value="0x0001">
-      <featureBit tag="LF" bit="0">true</featureBit>
-      <featureBit tag="TL" bit="1">false</featureBit>
-      <featureBit tag="PA" bit="2">false</featureBit>
-      <featureBit tag="ABS" bit="3">false</featureBit>
```

Comment:
zap was normally designed to handle feature map with this scheme

Reply:
not sure this style is still relevant today or not

Reply:
If I'm correctly informed this is outdated. I've added the "latest" version on how to reference the feature flags.